src/daos_vol.c

```diff
@@ -408,16 +399,13 @@ H5Pset_fapl_daos(hid_t fapl_id, const uuid_t pool_uuid, const char *pool_grp)
     } /* end if */
 
     /* Initialize driver specific properties */
-    uuid_copy(fa.pool_uuid, pool_uuid);
-
-    memset(fa.pool_group, '\0', sizeof(fa.pool_group));
-    if(pool_grp) {
-        size_t pool_grp_name_len = strlen(pool_grp);
+    strncpy(fa.pool, pool, DAOS_PROP_LABEL_MAX_LEN);
+    fa.pool[DAOS_PROP_LABEL_MAX_LEN] = 0;
 
-        if(pool_grp_name_len > H5_DAOS_MAX_GRP_NAME)
-            D_GOTO_ERROR(H5E_ARGS, H5E_BADVALUE, FAIL, "service group name is too long");
-        else if(pool_grp_name_len > 0)
-            strncpy(fa.pool_group, pool_grp, H5_DAOS_MAX_GRP_NAME - 1);
+    memset(fa.sys, '\0', sizeof(fa.sys));
+    if(sys_name) {
+	strncpy(fa.sys, sys_name, DAOS_SYS_NAME_MAX);
+	fa.sys[DAOS_SYS_NAME_MAX] = 0;
```

Comment:
@jhendersonHDF we should probably run clang-format on the daos vol source files after you're done with your changes